src/util/propertyTreeHelpers.ts

```diff
@@ -108,6 +108,12 @@ export function isRenderable(uri: Uri): boolean {
   return uri.endsWith('.Renderable');
 }
 
+export function isTransform(uri: Uri): boolean {
+  return (
+    uri.endsWith(ScaleKey) || uri.endsWith(TranslationKey) || uri.endsWith(RotationKey)
```

Comment:
I would prefer to make this check safer.  "Scale" is, for example, a common ending for many property owners that are not necessarily a transform. 

Could for example check that the URI includes two dots, i.e. that these keys are on the third level in the URI

Reply:
oculd maybe also update the funciton name to `isSgnTransform` if we want to make it even more clear, as well.